### DISCUSSION TITLE
TBATS keeps predicting the same Value

### DISCUSSION THREAD

**Discussion by aryan26roy** [alan-turing-institute/sktime#5243] [Q&A]
I am trying to use TBATS to predict sales data. The model, after training keeps predicting the same value for all the time steps (there is difference in decimal points but that goes away after rounding off the numbers since it's predicting sales of products). 

I looked into it and the value it is predicting is is the almost the same value as the 50% percentile for the training data.

Is there any way to break out of this? I have tried varying the length of training data (I have 356 time steps for training) but that doesn't work really well all the time. I need 7 or 14 time step prediction.

PS - I am running it without setting any 'use_*' arguments set as False and n_jobs=1.

**Comment by fkiraly** [comment]
> I looked into it and the value it is predicting is is the almost the same value as the 50% percentile for the training data.

This is the expected outcome if your data contains only noise, although it does not imply that this is the case.
Another possiility is intermittency or highly stochastic behaviour, in this case you might be more interested in prediction intervals than a point prediction (using `predict_interval`)

Of course, a bug is also a possibility, hard to tell without the actual data, or a sanitized dummy replicate that would allow to inspect what is going on.

**Comment by aryan26roy** [comment]
Thanks for the reply. I am trying to run predict_interval but both TBATS and ARIMA throw an error saying the listed models cannot predict intervals. How can I fix this? ( I can see in the docs that both ARIMA and TBATS are supposed to have an implementation for this method.

**Reply by aryan26roy** [reply]
It turns out it was a Python issue. These methods were not defined in the sktime version for Python 3.6.
